COCOAPODS.md

```diff
@@ -1,150 +1,381 @@
-and run an Xcode build.
+    target 'TVosApp' do
+      use_frameworks!
+      platform :tvos, '13.4'
+      
+      # Pods for TVosApp
+      pod 'kotlin_library', :path => '../kotlin-library'
+    end
+    ```
+    
+    </div>
+    
+6. Re-import the project.
 
-## Interoperability
+You can find a sample project [here](https://github.com/Kotlin/multitarget-xcode-with-kotlin-cocoapods-sample).
 
-The CocoaPods plugin also allows using CocoaPods libraries without manual configuring cinterop
-parameters (see the [corresponding section](https://kotlinlang.org/docs/reference/building-mpp-with-gradle.html#cinterop-support)
-of the multiplatform plugin documentation). The `cocoapods { ... }` code block allows you to
-add dependencies on CocoaPods libraries.
+### Add dependencies between multiple Kotlin Pods and an Xcode project
 
-<div class="sample" markdown="1" theme="idea" mode="kotlin">
+1. Create an Xcode project with `Podfile` if you haven’t done this yet.
+2. Create a hierarchical structure of your Kotlin projects - a root and child projects.
+3. Add the path to your Xcode project `Podfile` with `podfile = project.file(..)` to `build.gradle.kts` (`build.gradle`) of
+ the root project.  
+    This step helps synchronize your Xcode project with Kotlin Pod dependencies by calling `pod install` for your 
+    `Podfile`.
+4. If the root project doesn't represent a Kotlin Pod, specify that you don’t need a `Podspec` for this project - `noPodspec()`.
 
-```kotlin
-kotlin {
-    cocoapods {
-        // Configure a dependency on AFNetworking.
-        // The CocoaPods version notation is supported.
-        // The dependency will be added to all macOS and iOS targets.
-        pod("AFNetworking", "~> 3.2.0")
+    <div class="sample" markdown="1" theme="idea" data-highlight-only>
+    
+    ```kotlin
+    kotlin {
+        ios()
+        cocoapods {
+            noPodspec()
+            podfile = project.file("ios-app/Podfile")
+        }
     }
-}
-```
+    ```
+    
+    </div>
 
-</div>
+5. Add dependencies to Pod libraries that you want to use to the build script of your child projects using `pod()`.  
+    > Do NOT add a path to `Podfile` in your child projects.
+    {:.note}
+                                                                                      
+    <div class="sample" markdown="1" theme="idea" data-highlight-only>
+    
+    ```kotlin
+    kotlin {
+        ios()
+        cocoapods {
+            summary = "Working with external pods from Kotlin/Native using CocoaPods"
+            homepage = "https://github.com/JetBrains/kotlin-native"
+            pod("AFNetworking", "~> 4.0.0")
+        }
+    }
+    ```
+    
+    </div>
 
-To use these dependencies from a Kotlin code, import a package `cocoapods.<library-name>`.
-In the example above, it's `cocoapods.AFNetworking`.
+6. Add names and paths of the Kotlin Pods you want to include in the Xcode project to `Podfile`.
```

Comment:
6. Add the names and paths of the Kotlin Pods you want to include in the Xcode project to the `Podfile`.

Reply:
Removed the scenario